# print(x)

# # print("hello")
# try:
#     print(a)
#     print(a)
# except Exception as e
#     print(e)
# print("Hello")

try:
    num = int(input("Please enter a number: "))
    print(num)
except:
    print('Invalid entry, please try again!')